filters/include/pcl/filters/frustum_culling.h

```diff
@@ -144,14 +146,33 @@ namespace pcl
           throw PCLException ("Horizontal field of view should be between 0 and 180(excluded).",
             "frustum_culling.h", "setHorizontalFOV");
         }
-        hfov_ = hfov;
+        fov_left_bound_ = -hfov / 2;
+        fov_right_bound_ = hfov / 2;
+      }
+
+      /** \brief Set the horizontal field of view for the camera in degrees
```

Comment:
It might not be clear that the left bound should be (in most cases) given as a negative value. I think it would be good if the doc would mention that and also give an example, e.g. `setHorizontalFOV(60.0)` is equivalent to `setHorizontalFOV(-30.0, 30.0)`